diff -aurN pessulus-0.10.1.orig/data/pessulus.desktop.in.in pessulus-0.10.1/data/pessulus.desktop.in.in
--- pessulus-0.10.1.orig/data/pessulus.desktop.in.in	2006-03-18 08:43:11.000000000 +0100
+++ pessulus-0.10.1/data/pessulus.desktop.in.in	2006-06-12 23:09:16.918013560 +0200
@@ -11,5 +11,5 @@
 X-GNOME-Bugzilla-Product=pessulus
 X-GNOME-Bugzilla-Component=general
 X-GNOME-Bugzilla-Version=@VERSION@
-Categories=GNOME;Application;System;Settings;
+Categories=GTK;GNOME;Settings;
 OnlyShowIn=GNOME;
